common/mixnet-contract/src/lib.rs

```diff
@@ -18,6 +18,17 @@ pub struct LayerDistribution {
     pub invalid: u64,
```

Comment:
So are we not removing it? : )

Reply:
Yes we are :) Just one more (tiny) commit :)